net/nginx-util/src/nginx-ssl-util.cpp

```diff
@@ -0,0 +1,658 @@
+#include <thread>
+
+#ifdef NO_PCRE
+#include <regex>
+namespace rgx = std;
+#else
+#include "regex-pcre.hpp"
+#endif
+
+#include "nginx-util.hpp"
+#include "px5g-openssl.hpp"
+
+
+#ifndef NO_UBUS
+static constexpr auto UBUS_TIMEOUT = 1000;
+#endif
+
+// once a year:
+static constexpr auto CRON_INTERVAL = std::string_view{"3 3 12 12 *"};
```

Comment:
fuchsia-statically-constructed-objects warns about static here. Correct me if I'm wrong, but static on a constexpr function is useless, no?

Reply:
There can be a difference between static constexpr and constexpr in general as explained at [stackoverflow](https://stackoverflow.com/questions/13865842/does-static-constexpr-variable-inside-a-function-make-sense). But, here it is maybe pointless. Would it be better to remove it? I will remove it at least for the literals _server_name and co. They exist only for using them in the template …